pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -958,7 +965,13 @@ func (p *Provider) Create(
 		if opts.GeoDistributed {
 			zones = defaultZones
 		} else {
-			zones = []string{defaultZones[0]}
+			// N.B. We default to the first two regions, namely us-east1 and us-central1. In order to balance the quotas,
+			// we place ~75% of the cluster in us-east1 and ~25% in us-central1.
+			if rand.Float64() < 0.75 {
+				zones = []string{defaultZones[0]}
+			} else {
+				zones = []string{defaultZones[1]}
```

Comment:
This won't be great if we get unlucky and run a test that takes large backups and stores them in our backup testing buckets, which are only in us-east1 (not multiregion).